src/System/FeatureFlags/unleashClient.ts

```diff
@@ -0,0 +1,27 @@
+import { getENV } from "Utils/getENV"
+import { UnleashClient } from "unleash-proxy-client"
+
+let unleashClient: UnleashClient | null = null
+
+export function getOrInitUnleashClient(): UnleashClient {
+  if (unleashClient) {
+    return unleashClient
+  }
+
+  const config = {
+    refreshInterval: 30, // How often (in seconds) the client should poll for updates
+    url: `${getENV("UNLEASH_API_URL")}/frontend`,
+    clientKey: getENV("UNLEASH_FRONTEND_KEY"),
+    appName: getENV("UNLEASH_APP_NAME"),
+    environment: getENV("UNLEASH_ENVIRONMENT"),
+    context: {
+      userId: getENV("CURRENT_USER")?.id,
+      sessionId: getENV("SESSION_ID"),
+    },
```

Comment:
Didn't realize you could use `getENV()` this way

Reply:
As long as its not top-level and is in a function or something (meaning, there's some instantiation that happens on app boot) then its fine. Its how we access sharify values isomorphically.